### PR TITLE
Fix typos in comments of src/utils.

### PR DESCRIPTION
**Pull request by dongjoon-hyun**

### CODE REVIEW

File: src/components/ContainerDetailsSubheader.react.js [hunk 1]
```diff
@@ -99,7 +99,7 @@ var ContainerDetailsSubheader = React.createClass({
   },
   handleTerminal: function () {
     if (!this.disableTerminal()) {
-      metrics.track('Terminaled Into Container');
+      metrics.track('Terminated Into Container');
```
Comment (FrenchBen) [inline src/components/ContainerDetailsSubheader.react.js]: Although it looks like a typo, this is actually correct, as we opened a 'terminal into the container', hence the use of 'Terminaled' as a verb

Comment (dongjoon-hyun) [inline src/components/ContainerDetailsSubheader.react.js]: Oops, sorry. I'll remove this.

**Comment by dongjoon-hyun** [comment]
I squashed the commits into a single one and changed the title, too.

**Comment by FrenchBen** [comment]
LGTM 
@jeffdm PTAL

**Comment by jeffdm** [comment]
LGTM! Thanks!

**Comment by dongjoon-hyun** [comment]
Thank you for merging!

**Pull request merged**